specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2023-11-01/ContentPackages.json

```diff
@@ -0,0 +1,319 @@
+{
```

Comment:
Readme file is missing in your PR, with these new swagger files added to it. That is why it shows Avocado errors also in your PR.

Reply:
Thanks. Updated